code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -463,7 +461,7 @@
 		..()
 
 /mob/living/simple_animal/update_mobility(value_otherwise = TRUE)
-	if(IsUnconscious() || IsParalyzed() || IsStun() || IsKnockdown() || stat || resting)
+	if(HAS_TRAIT(src, TRAIT_KNOCKEDOUT) || IsParalyzed() || IsStun() || IsKnockdown() || stat || resting)
```

Comment:
This should also check stat rather than knocked out

Reply:
Done in 4fc03d8